.github/workflows/nvidia-test.yml

```diff
@@ -51,3 +51,40 @@ jobs:
         with:
           poll: true
           job-path: ${{ env.TESTFLINGER_DIR }}/uc-nvidia-job.yml
+
+  test:
+    runs-on: [self-hosted, self-hosted-linux-amd64-jammy-private-endpoint-medium]
+    strategy:
+      matrix:
+        track: ${{ fromJSON(inputs.snap-tracks || '["latest/edge"]') }}
+        distro:
+          - noble
+    env:
+      TESTFLINGER_DIR: .github/workflows/testflinger
+      SNAP_CHANNEL: ${{ matrix.track }}
+    steps:
+      - name: Event data
+        run: "echo ::notice::Snap channel: $SNAP_CHANNEL"
+
+      - name: Checkout code
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
+      - name: Bypass aproxy for testflinger.canonical.com host
+        uses: ./.github/actions/set-aproxy-bypass-for-testflinger
```

Comment:
As hinted by the testflinger folks, this bit is no longer needed as it's integrated in the main `submit` action they maintain.

I've tweaked the existing job in https://github.com/canonical/lxd-ci/pull/478

Reply:
cool, please merge #478 then. I'll rebase this thing later.